unic/utils/src/traits.rs

```diff
@@ -0,0 +1,21 @@
+//! # Unic - Utils - Traits
+//!
+//! A component of [`unic`: Unicode and Internationalization Crates for Rust](/unic/).
+//!
+//! Defines traits used extensively across UNIC.
+
+use std::{fmt, str};
+
+/// A Property that applies to a character
+pub trait CharProperty : str::FromStr + fmt::Display {
```

Comment:
The `CharProperty` in this PR is merely a placeholder for the proper trait(s). I fully expect to wait on your traits and then to migrate to implementing those as appropriate rather than the ad-hoc way it is currently defined.